lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -961,7 +961,7 @@ def ng_update_session_flows(
                 if ret:
                     # Install the Rules
                     failed_policy_rule_results =\
-                        self._ng_qer_update(request, pdr_entries)
+        self._ng_qer_update(request, pdr_entries)
```

Comment:
This looks a bit strange and it seems it's breaking the Python formatting check. Was it intentional?

Reply:
No, this was not intentional. Thank you :+1: